build/README.md

```diff
@@ -1,31 +1,31 @@
 # Building Embassy OS
 
- The commands given assume a Debian or Ubuntu-based environment.  Variables that *MUST* be changed are denoted by `{THING}`. Replace this in its entirety; in other words, there will not be any `{}`s remaining after you enter your variable.
+ The commands given assume a Debian or Ubuntu-based environment.
```

Comment:
Add the warning or attention emoji to both the beginning and end of this line. I guarantee you no one will read this because people are notoriously bad at reading things that aren't in the direct bulleted list.

Alternatively make it part of the step by step instructions to "acquire a debian/ubuntu build environment"